pkg/retriever/reportretriever.go

```diff
@@ -104,70 +108,72 @@ func (r *Retriever) StartTokenRefresh() error {
 	return nil
 }
 
-func (r *Retriever) RetrieveCCXReport(input chan string, output chan types.PolicyInfo) {
+// RetrieveCCXReport ...
+func (r *Retriever) RetrieveCCXReport(input chan types.ManagedClusterInfo, output chan types.PolicyInfo) {
 	for {
-
-		clusterId := <-input
+		cluster := <-input
 		// If the cluster id is empty do nothing
-		if clusterId == "" {
+		if cluster.Namespace == "" || cluster.ClusterID == "" {
 			return
-		} else {
-			glog.Infof("RetrieveCCXReport for cluster %s", clusterId)
 		}
-		req, err := r.GetInsightsRequest(context.TODO(), r.CCXUrl, clusterId)
+
+		glog.Infof("RetrieveCCXReport for cluster %s", cluster.Namespace)
+		req, err := r.GetInsightsRequest(context.TODO(), r.CCXUrl, cluster)
 		if err != nil {
-			glog.Warningf("Error creating HttpRequest for cluster %s, %v", clusterId, err)
+			glog.Warningf("Error creating HttpRequest for cluster %s, %v", cluster.Namespace, err)
```

Comment:
We use the cluster Id to get the data - I am thinking having cluster ID logged is more useful than the cluster name.